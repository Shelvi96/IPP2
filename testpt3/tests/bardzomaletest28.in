NEW ID

::>111
1;181:00:9>70989;:
:0926>4:064:73
48:810>17123:25:4
8;7736092:>9956;57;20
7>159:
58>43
817975696;>:421:205
52>4:845
;00572048>30458667
:42875>5
@:5;364:09706:03
@111024490;24874;032:84:5;;39
@6440
@42213249:54
@495
@;122:256384
@54:1;0626;04486
@9;:950;
@285:8:616475850244:0
@44:72;03209:576350:1;7947
@1;11496650
@97;959343:2703685;700;65733184
@:;978:2511715271;61064:4;1702
@2033;3
@286777023611750;34:9;23387
@;61002:7968;12529
@206623:4:50
@33;43065;8;770903:39:9440602;4
@27:62258217900717944050496
