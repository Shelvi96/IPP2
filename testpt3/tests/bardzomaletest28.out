90
4594947029268288
0
0
0
0
82
0
4842748
3452035744388
0
2420294228500414464
46367169152535517
0
116719436173824
10110
0
551431618475888557
10111001000000
